code/modules/materials/materials.dm

```diff
@@ -745,11 +745,11 @@ var/list/name_to_material
 	hardness = 1
 	weight = 1
 	protectiveness = 0 // 0%
-	stack_origin_tech = list(TECH_MATERIAL = 1)
+	stack_origin_tech = list(TECH_MATERIAL = 2) //Snow seems more complex than cardboard.

```

Comment:
Let's not do that. Leave it at 1, please.

Reply:
Alright.. I mean.. You can disassemble a simple cardboard box and get the same amount of research.

Reply:
From a R&D perspective, neither mats 1 nor mats 2 will be at all relevant tbh nobody's going out of their way to find some snow to deconstruct when they're deconning an advanced mining drill anyway

Reply:
I doubt snow is more complex than cardboard in terms of this research.

Reply:
Every snowflake is unique. Not exactly in viewing terms, but NO snowflake has the same combination of atoms, atom structure, or size put together.

Reply:
Either way, I'm knocking it back down to one.

Reply:
no cardboard box has the same combination of atoms, atom structure, or size put together